example_configs/zookeeper.json

```diff
@@ -0,0 +1,55 @@
+{
+  "lowercaseOutputName": true,
+  "lowercaseOutputLabelNames": true,
+  "rules": [
+    {
+      "pattern": "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\d)><>(\w+)",
+      "name": "zookeeper_$2",
+      "attrNameSnakeCase": true,
+      "labels": {
+        "brokerId": "$1"
+      }
+    },
+    {
+      "pattern": "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\d), name1=replica.(\d)><>(\w+)",
+      "name": "zookeeper_$3",
+      "attrNameSnakeCase": true,
+      "labels": {
+        "brokerId": "$1",
+        "replicaId": "$2"
+      }
+    },
+    {
+      "pattern": "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\d), name1=replica.(\d), name2=(\w+)><>(\w+)",
+      "name": "zookeeper_$4",
+      "attrNameSnakeCase": true,
+      "labels": {
+        "brokerId": "$1",
+        "replicaId": "$2",
+        "memberType": "$3"
+      }
+    },
+    {
+      "pattern": "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\d), name1=replica.(\d), name2=(\w), name3=(\w)><>(\w+)",
+      "name": "zookeeper_$4_$5",
+      "attrNameSnakeCase": true,
+      "labels": {
+        "brokerId": "$1",
+        "replicaId": "$2",
+        "memberType": "$3"
+      }
+    },
+    {
+      "pattern": "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\d), name1=replica.(\d), name2=(\w+), name3=(\w+), name4=(.+), name5=(.+)><>(\w+)",
+      "name": "zookeeper_$4_$7",
+      "attrNameSnakeCase": true,
+      "labels": {
+        "brokerId": "$1",
+        "replicaId": "$2",
+        "memberType": "$3",
+        "ipAddress": "$5",
+        "hexSessionId": "$6"
```

Comment:
Listing every session is going to get very big, this shouldn't be exposed by default